string tag
float32 confidence
